modules/pytket-honeywell/tests/backend_test.py

```diff
@@ -51,7 +51,7 @@
 
 def test_honeywell() -> None:
     backend = HoneywellBackend(
-        device_name="HQS-LT-1.0-APIVAL", machine_debug=skip_remote_tests
+        device_name="HQS-LT-S1-APIVAL", machine_debug=skip_remote_tests
```

Comment:
What is the difference?

Reply:
the previous is an older device, which is no longer available for testing

Reply:
ok